docs/source/deploying-volttron/multi-platform/multi-platform-rabbitmq-deployment.rst

```diff
@@ -260,13 +249,22 @@ upstream servers on the downstream server and make the VOLTTRON exchange
 
            vctl rabbitmq list-federation-parameters
            NAME                         URI
-           upstream-volttron2-rabbit-2  amqps://rabbit-2:5671/volttron2?cacertfile=/home/nidd494/.volttron1/certificates/certs/volttron1-root-ca.crt&certfile=/home/nidd494/.volttron1/certificates/certs/volttron1-admin.crt&keyfile=/home/nidd494/.volttron1/certificates/private/volttron1-admin.pem&verify=verify_peer&fail_if_no_peer_cert=true&auth_mechanism=external&server_name_indication=rabbit-2
+           upstream-volttron2-volttron amqps://volttron2:5671/volttron?cacertfile=/home/volttron/vhome/test_fed/certificates/federation/volttron2_ca.crt&certfile=/home/volttron/vhome/test_fed/certificates/federation/volttron2.volttron1.federation.crt&keyfile=/home/volttron/vhome/test_fed/certificates/private/volttron1.federation.pem&verify=verify_peer&fail_if_no_peer_cert=true&auth_mechanism=external&server_name_indication=volttron2
+
 
      Copy the upstream link name and run the below command to remove it.
 
        .. code-block:: bash
 
-         vctl rabbitmq remove-federation-parameters upstream-volttron2-rabbit-2
+         vctl rabbitmq remove-federation-parameters upstream-volttron2-volttron
+
+.. note::
+
+    These commands only remove the federation parameter from RabbitMQ and certificate entries from rabbitmq_federation_config.yml on the publisher node.
+    `It does not remove the actual certificates.` If you need to rerun the federation command again for the same setup and need to create fresh certificates, then you will
+    need to manually remove public and private certificates. Private certificates will be in
```

Comment:
This is not sufficient. If the user really wants new certificate, then they should request the remote instance admin to delete there earlier generated cert before a new CSR is sent for approval

Reply:
done